index.js

```diff
@@ -0,0 +1,60 @@
+
+var canvas = document.querySelector("canvas"), 
+ctx = canvas.getContext("2d");
+
+function draw(n,data){
```

Comment:
Please keep in mind rules about code formatting, currently it's not really readable:
`draw(n, data)`
`} else if (data == 'data/32x32.json') {`
`let width = n;
 let height = n;
 let scale = canvas.width / n;`
logical separation blocks of code,
`makeColor = function(d) {
     return '#' + d;
};` <--- this form is much easier for debugging.